src/main/java/graphql/analysis/NodeVisitorWithTypeTracking.java

```diff
@@ -3,8 +3,8 @@
 import graphql.GraphQLContext;
```

Comment:
Update file to bring in line with v21 - it's because ConditionalNodes are referenced in the ExecutableNormalizedOperationFactory, and those changes are viral in a few other files